examples/reference/layouts/Tabs.ipynb

```diff
@@ -135,7 +137,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "tabs = pn.Tabs(p1, p2, p3)\n",
+    "tabs = pn.Tabs(p1, p2, p3, dynamic=True)\n",
```

Comment:
I'm -1 on changing this here, both `dynamic=True` and `closable=True/False` need a little section that explains what they do.

Reply:
Sorry Philipp, I don't get it. You mean using these both parameters in there like;
'"tabs = pn.Tabs(p1, p2, p3, dynamic=True, closable=True)\n", '?

Reply:
No, I mean adding a completely new section for each of these and describing in detail what they do.